zebra/zebra_vxlan.c

```diff
@@ -285,7 +285,8 @@ static void zvni_find_neigh_addr_width(struct hash_backet *backet, void *ctxt)
 
 	n = (zebra_neigh_t *)backet->data;
 
-	ipaddr2str(&n->ip, buf, sizeof(buf)), width = strlen(buf);
+	ipaddr2str(&n->ip, buf, sizeof(buf));
+	width = strlen(buf);
```

Comment:
these lines should be reversed and width passed to ipaddr2str

Reply:
@louberger Hi, unless I missed something, it should be ok as it is (the string length of 'buf' is only known after the ipaddr2str() call) (?)

P.S. the change does not change the code behavior.